source/windows/thread.c

```diff
@@ -89,5 +111,5 @@ void aws_thread_current_sleep(uint64_t nanos) {
     /* We don't really have a better option here for windows that isn't super complex AND we don't have a use case
      * yet where we should have sleeps anywhere other than for context switches and testing. When that time arises
      * put the effort in here. */
-    Sleep(nanos / 1000000);
+    Sleep((DWORD)(nanos / 1000000));
```

Comment:
Is this cast needed?

Reply:
yeah, Sleep takes DWORD, nanos / 1000000 is uint64_t